package.json

```diff
@@ -6,6 +6,9 @@
   "scripts": {
     "start": "pm2 start process.json",
     "dev": "pm2 start process-dev.json",
+    "local": "nodemon ./dist/src/app.js",
```

Comment:
lets not commit `/dist` and add it to `gitignore`

Reply:
Added to gitignore